packages/middleware-logger/src/logger.util.ts

```diff
@@ -0,0 +1,29 @@
+import { WriteStream } from 'fs';
+import { HttpResponse } from '@marblejs/core';
+import { Maybe, compose } from '@marblejs/core/dist/+internal';
+import { LoggerOptions } from './logger.model';
+
+export const isNotSilent = (opts: LoggerOptions) => () =>
+  !opts.silent;
+
+export const filterResponse = (opts: LoggerOptions) => (res: HttpResponse) =>
+  Maybe.of(opts.filter)
+    .map(filter => filter(res))
+    .valueOr(true);
+
+export const writeToStream = (stream: WriteStream, chunk: string) =>
+  stream.write(`${chunk}\n\n`);
+
+export const formatTime = (timeMs: number) =>
+  timeMs > 1000
+    ? `${timeMs / 1000}s`
+    : `${timeMs}ms`;
+
+export const getTimeDifferencIneMs = (startTime: Date): number =>
```

Comment:
Shouldn't it be named `getTimeDifferenceInMms` 😎 ?

Reply:
Or just `getTimeDifferenceMs()`

Reply:
@krzysztof-miemiec IMO `getTimeDifferenceInMs` is more descriptive but it is a cosmetic type issue ✌️